dotCMS/src/integration-test/java/com/dotcms/content/elasticsearch/business/ESContentletAPIImplTest.java

```diff
@@ -703,6 +704,89 @@ public void testCheckInArchivedPageShouldThrowDotDataException() {
 
     }
 
+    /**
+     * Method to test: {@link ESContentletAPIImpl#move(Contentlet, User, Host, Folder, boolean)}
+     * Given Scenario: sends a null host and folder path
+     * ExpectedResult: The method should not throw a {@link IllegalArgumentException}
```

Comment:
Please update the doc of all tests included here, as it seems not to be totally related to what tests do or expect as result

Reply:
actually the description is accurate which the test